server/plugin/api.go

```
@@ -196,6 +200,199 @@ func (p *Plugin) handleUserDisconnect(w http.ResponseWriter, r *http.Request) {
 	p.returnPostActionIntegrationResponse(w, response)
 }
 
+func (p *Plugin) handleDateTimeSelectionDialog(w http.ResponseWriter, r *http.Request) {
+	decoder := json.NewDecoder(r.Body)
+	postActionIntegrationRequest := &model.PostActionIntegrationRequest{}
+	if err := decoder.Decode(&postActionIntegrationRequest); err != nil {
+		p.API.LogError("Error decoding PostActionIntegrationRequest params.", "Error", err.Error())
+		return
```

don't we need to return the error here?